doc/usage/restructuredtext/field-lists.rst

```diff
@@ -45,3 +45,12 @@ At the moment, these metadata fields are recognized:
        :orphan:
 
    .. versionadded:: 1.0
+
+``nosearch``
```

Comment:
I have another candidate: `:search: no`. But I chose `nosearch` because it is similar to existing metadata: `nocomments`.

Reply:
+1 for `nosearch`